### DISCUSSION TITLE
v0.18.0 draft

### DISCUSSION THREAD

**Discussion by Conengmo** [python-visualization/folium#2015] [Collaborators]
Hi @ocefpaf @hansthen,

I made a draft release of v0.18.0, could one of you take a look see if it's good to go?

https://github.com/python-visualization/folium/releases

**Comment by ocefpaf** [comment]
I would remove the bot PR in the notes

`docs: Update switcher.json for v0.17.0 by @github-actions in https://github.com/python-visualization/folium/pull/1973`

but the rest is good IMO.

**Reply by Conengmo** [reply]
good catch! Forgot that one.

**Reply by Conengmo** [reply]
Alright let's go then! 🚀

**Reply by ocefpaf** [reply]
Frank, I just noticed that our bus factor on PyPI is quite low. It is only me and Rob, which means it is only me. Do you and/or @hansthen have a PyPI user handle so I can you?

**Reply by hansthen** [reply]
Mine is `hans.then`

**Reply by ocefpaf** [reply]
Invitation sent!

**Reply by Conengmo** [reply]
Good call! I'm 'Conengmo' on Pypi.

**Reply by ocefpaf** [reply]
Invitation sent!
